src/bin/dashboard_src/dashboard_app.rs

```diff
@@ -266,20 +273,25 @@ impl DashboardApp {
                     console_queue.remove(0);
                 }
                 drop(console_queue);
+                if draw {
+                    terminal.draw(|f| app.render(f))?;
+                }
             }
 
             if let Some(string) = console_input {
-                app.handle(DashboardEvent::ConsoleMode(ConsoleIO::InputSupplied(
-                    string,
-                )))
+                app.handle(
+                    &mut terminal,
+                    DashboardEvent::ConsoleMode(ConsoleIO::InputSupplied(string)),
+                )
                 .await?;
             }
 
-            terminal.draw(|f| app.render(f))?;
-
+            // note: setting a low duration like 100 can cause high CPU usage
```

Comment:
What does 100 mean here (in the comment), microseconds, ticks?

Reply:
milliseconds.  so 100 = 1/10 second.

The comment applies to the following line is, which calls `::from_millis()`
`if event::poll(Duration::from_millis(100))?`